test/transactions/vote.spec.ts

```diff
@@ -1,10 +1,10 @@
 import {VoteAsset, VoteTx} from '../../src/trxTypes/Vote'
 import {TransactionType} from 'risejs';
 import {expect} from 'chai';
-import {BaseTx, Transaction} from "../../src/trxTypes/BaseTx";
+import {BaseTx, Transaction} from '../../src/trxTypes/BaseTx';
 import {testPrivKey} from '../testConsts';
 
-const txs        = require(`${__dirname}/../data/voteTxs.json`);
```

Comment:
@Isabello this wont work. `` are used for template literals -> https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals